spec/lib/mailchimp_spec.rb

```diff
@@ -93,12 +93,6 @@
 			expect(MailchimpNonprofitUserAddJob).to have_been_enqueued.with(drip_email_list, nonprofit_user, nonprofit_user.roles.first.host)
  		end 
 
-		# it 'will NOT include users that doesnt belong to a nonprofit' do 
-		# 	ActiveJob::Base.queue_adapter = :test
-		# 	Mailchimp.sync_nonprofit_users(drip_email_list)
-		# 	expect(MailchimpNonprofitUserAddJob).to_not have_been_enqueued.with(user)
-		# end 
-
```

Comment:
We do need to test this, we didn't have an opportunity to go through the testing however. During our pairing session. I can do work on that today, I think.